docs/content/guides/dagster/asset-tutorial/testing-assets.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Testing Assets | Dagster
+description: Dagster enables you to unit-test individual assets and graphs of assets
+---
+
+# Testing Assets
+
+Creating testable and verifiable data pipelines is one of the focuses of Dagster. We believe ensuring data quality is critical for managing the complexity of data systems. Here, we'll cover how to write unit tests for individual assets, as well as for graphs of assets together.
+
+## Testing the Cereal Asset Definitions
+
+Let's go back to the assets we defined in the [prior section](/guides/dagster/asset-tutorial/asset-graph#a-more-complex-asset-graph), and ensure that they work as expected by writing some unit tests.
+
+We'll start by writing a test for the `nabisco_cereals` asset definition, which filters the larger list of cereals down to the those that were manufactured by Nabisco. To run the op (function) that derives an asset from its upstream dependencies, we can invoke it directly, as if it's a regular Python function:
```

Comment:
I think it would be nice if we didn't have to introduce "op" here as another noun in the tutorial, especially when it doesn't really add to the understanding of an asset here. 

We could just say: To run the asset, we can invoke it directly....

Reply:
I think you're right about not introducing ops - will get rid of that.  I do think there's an important distinction between the "asset" and the "function that derives the asset from its upstream dependencies", so prefer to keep that part in.